### PR TITLE
major updates

### PR DESCRIPTION
**Pull request by fizzka**
* changed build to github-actions (mainly uses matrix build)
* added php8.2 support
* added arm64 support =)
* updated phalcon to 5.3.1
* cleaned obsolete stuff

please, approve PR, so I'll change paths and labels to your repo.

u have to add action secrets to access docker hub

link to already generated images: https://hub.docker.com/r/fizzka/docker-phalcon/tags

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: ci
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  IMAGE_NAME: fizzka/docker-phalcon
```
Comment (MilesChou) [inline .github/workflows/build.yml]: Image name: `mileschou/phalcon` / `fizzka/docker-phalcon`

I'm not sure what which one to choose.

I guess that my secret access token just can update to my dockerhub repo.

Comment (fizzka) [reply .github/workflows/build.yml]: "so I'll change paths and labels to your repo." - please look before, after I commit fixes to your repos =)

Comment (fizzka) [reply .github/workflows/build.yml]: I think, you should generate new token for github actions
and put it here:

secrets (r&w required):
image

Comment (fizzka) [reply .github/workflows/build.yml]: UP

Comment (MilesChou) [reply .github/workflows/build.yml]: "generate new token for github actions and put it"

Ready to go.

**Review by MilesChou — APPROVED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (fizzka) [reply .github/workflows/build.yml]: Did you create token, so I will change image name?

**Pull request merged**